term/src/terminalstate/performer.rs

```diff
@@ -132,7 +132,7 @@ impl<'a> Performer<'a> {
         for g in Graphemes::new(text) {
             let g = self.remap_grapheme(g);
 
-            let print_width = grapheme_column_width(g, Some(self.unicode_version));
+            let print_width = grapheme_column_width(g, Some(self.unicode_version.clone()));
```

Comment:
re: performance anxiety, this is a very hot loop, especially for folks doing `cat bigfile`.

I think we should be passing `Some(&self.unicode_version)` here to avoid any additional overhead.

There will be more API fan-out, but I think it is worth it to keep the performance reasonable.

The termwiz Cargo.toml version number would also need to be bumped because it would be a breaking API change, even though it is a minor one.